### PR TITLE
R92

### PR DESCRIPTION
**Pull request by BenFradet**
- [x] changelog rel date
- [x] release lock if no-op
- [x] removal of rdb loader tmp files
- [x] rdb logs log level
- [x] rdb logs when cancelled
- [x] archive shredded

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,8 +35,6 @@ env:
   matrix:
   - TEST_DIR=3-enrich/scala-common-enrich/
   - TEST_DIR=3-enrich/spark-enrich/
-  - TEST_DIR=4-storage/rdb-shredder/
```
Comment (chuwy) [inline .travis.yml]: Links in `4-storage/README.md` still point to projects in `snowplow/snowplow`.

File: 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb [hunk 2]
```diff
@@ -473,17 +473,22 @@ def initialize(debug, staging, enrich, shred, es, archive_raw, rdb_load, archive
           end
         end
 
+        latest_run_id = get_latest_run_id(s3, csbe[:good])
         if archive_enriched == 'pipeline'
-          archive_enriched_step = get_archive_enriched_step(csbe[:good], csbe[:archive], run_id, s3_endpoint, ": Enriched S3 -> Enriched Archive S3")
+          archive_enriched_step = get_enriched_step(csbe[:good], csbe[:archive], run_id, s3_endpoint, ": Enriched S3 -> Enriched Archive S3")
```
Comment (chuwy) [inline 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb]: Not sure `get_enriched_step` exists.

File: 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb [hunk 3]
```diff
@@ -473,17 +473,22 @@ def initialize(debug, staging, enrich, shred, es, archive_raw, rdb_load, archive
           end
         end
 
+        latest_run_id = get_latest_run_id(s3, csbe[:good])
```
Comment (chuwy) [inline 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb]: I think it is hypothetically possible to have different run ids in `shredded` and `enriched`. Therefore `latest_run_id` must be evaluated for both steps separately (and only if we're in `recovery`)

Comment (BenFradet) [inline 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb]: yup this is done locally

File: 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb (hunk 2 above)
Comment (BenFradet) [inline 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb]: fixed locally too

File: .travis.yml (hunk 1 above)
Comment (BenFradet) [inline .travis.yml]: will fix

**Comment by BenFradet** [comment]
@alexanderdean I put down tomorrow as the release date, if a release doesn't happen tomorrow that'll need to be changed.

What's left to be done:

- rm the rc commits (I left them in case there is need for another one)
- merge
- tags

**Comment by alexanderdean** [comment]
Please do this release on Monday @BenFradet !

**Pull request merged**